src/applications/ivc-champva/10-7959f-2/config/form.js

```diff
@@ -27,10 +29,16 @@ import {
   internationalPhoneUI,
 } from '../../shared/components/InternationalPhone';
 import PaymentSelectionUI from '../components/PaymentSelection';
+import { fileUploadUi as fileUploadUI } from '../../shared/components/fileUploads/upload';
+import { UploadDocuments } from '../components/UploadDocuments';
 
 const veteranFullNameUI = cloneDeep(fullNameUI());
 veteranFullNameUI.middle['ui:title'] = 'Middle initial';
 
+function FileFieldWrapped(props) {
+  return FileFieldCustom({ ...props, requiredFiles: [] });
+}
```

Comment:
If we don't have to use `FileFieldCustom` we should be able to skip this (and using a custom page for the file upload) as well